app/models/concerns/thredded/post_common.rb

```diff
@@ -13,14 +13,18 @@ module PostCommon
 
       validates :content, presence: true
 
-      scope :order_oldest_first, -> { order(id: :asc) }
-      scope :order_newest_first, -> { order(id: :desc) }
+      scope :order_oldest_first, -> { order(created_at: :asc) }
```

Comment:
There will be clashes in integration tests, leading to hard to debug flakes. Actually, thinking about it, this might be the reason for #352!

And I've seen clashes like this in production on MySQL, it's not that unlikely because [birthday problem](https://en.wikipedia.org/wiki/Birthday_problem).